docs/source/index.rst

```diff
@@ -0,0 +1,22 @@
+ytdl-sub User Guide
+===================
+
+.. toctree::
+  :maxdepth: 2
+  :titlesonly:
+
+  introduction
+  guides/index
+  prebuilt_presets/index
+  usage
+  config_reference/index
+  faq/index
+  deprecation_notices
+
+.. note:: End goal similar to: https://picard-docs.musicbrainz.org/en/functions/list_by_type.html
```

Comment:
Let's remove this

Reply:
Or better yet make like a two sentence statement about how the docs are an active WIP